services/graylog/Makefile

```diff
@@ -56,3 +60,8 @@ clean: ## Cleans unversioned files
 ${TEMP_COMPOSE}: docker-compose.yml
 	@docker-compose -f $< config > $@
 	@echo "${STACK_NAME} stack file created for in $@"
+
+.PHONY: ${TEMP_COMPOSE}-letsencrypt
+${TEMP_COMPOSE}-letsencrypt: docker-compose.yml docker-compose.letsencrypt.yml
```

Comment:
Instead of variable ``TEMP_COMPOSE`` I started to create a fixed value for this output and add it in the gitignore list. It helps to debug if something goes wrong. 

An example is [.stack-simcore-development.yml](https://github.com/ITISFoundation/osparc-simcore/blob/master/Makefile#L133). It is an output of make in osparc-simcore and compresses

Reply:
yes in this case this one converts to .stack.graylog-letsencrypt.yml. I actually prefer using an automatic variable that also scales better when you create other services instead of renaming all the recipes.
does that make sense or I missed the point?

Reply:
My bad. I misinterpreted from the partial file the implementation of ``${TEMP_COMPOSE}``. It does exactly what I meant. I got confused by the name ``${TEMP_COMPOSE}`` since I thought it was created randomly everytime. Perhaps ``letsencrypt-${TEMP_COMPOSE}`` so that it has ``yml`` extension?